addons/account/tests/test_account_all_l10n.py

```diff
@@ -23,12 +22,18 @@ def test_all_l10n(env):
     env.reset()     # clear the set of environments
     env = env()     # get an environment that refers to the new registry
 
-    # Install the demo data when testing
-    env.ref('base.module_account').demo = True
-    coas = env['account.chart.template'].search([])
+    coas = env['account.chart.template'].search([
+        ('id', 'not in', env['res.company'].search([]).chart_template_id.ids)
+    ])
     for coa in coas:
         cname = 'company_%s' % str(coa.id)
-        company = env['res.company'].create({'name': cname})
+        company = env['res.company'].create({
+            'name': cname,
+            'country_id': env['res.company'].sudo().browse(env['ir.model.data'].search([
+                ('model', '=', 'res.company'),
+                ('module', '=', coa.get_metadata()[0]['xmlid'].split('.')[0])
```

Comment:
Need to see if we can use country_id on the coa instead.  (syscohada will remain difficult)